_pages/en/strategy-open-first.md

```diff
@@ -0,0 +1,445 @@
+It is also to provide a list of recommendations, considerations and procedures for the adoption, management and promotion of OSS as the primary option for future and next iteration solutions of ESDC/IITB information systems.
+Some short term recommendations are:
+
+- Define and focus on the preferred areas of adoption, consider OSS in every aspect of the ICT Ecosystem+;
+- Implement a process for the sharing and re-use of developed OSS efforts within the department, GoC and OSS Community; and
+- Establish an OSS Program Office to take on the responsibility and further define the process and reporting requirements.
+
+## 4 Goals and Outcomes
+
+These days, open standards and OSS is everywhere.
+It has reshaped the landscape in the IT industry, is ubiquitous in manufacturing, and is an increasingly common instrument of public services.
+Governments across Canada and in the worlds' leading nations have adopted policies on open standards and OSS.
+
+Properly implemented, the strategy, its governing principles and its action plan will help us build and deliver better ICT solutions and services, to leverage the innovative and collaborative power of open standards, OSS and working in the open.
+They will make ESDC and the GC part of the OSS community, an agile contributor and active participant.
+By co-developing with other Canadian public-sector organisations and encouraging the sharing and reuse of our solutions, knowledge and expertise, we will deliver better services that enrich society and reduce its costs.
+
+The strategy focuses on transformative and innovative aspects, and recognises open standards and OSS as catalysts for change.
+In addition, it leverages initiatives in other GC departments/agencies and helps shape the conditions for OSS development and government innovation.
+This chapter sets out the key reasons and principles underlying the new strategy.
+
+### Progress towards digital autonomy
+
+Change is a constant the department has to continuously take into account when acquiring and developing software based solutions to offer services to its citizens.
+
+One high-level argument for an emphasis on open standards and OSS is made in the Prime Minister's mandate letter to the Minister of Digital Government:
+
+> Encourage the use and development of OSS products and open data, allowing for experimentation within existing policy directives and building an inventory of validated and secure applications that can be used by government to share knowledge and expertise to support innovation.
+
+Open standards and OSS impact the digital autonomy of Canada.
+Against the hyper-scalers in the cloud, it is likely that open standards and OSS can give ESDC and the GC a chance to create and maintain its own, independent digital approach and stay in control of its processes, its information and its technology, because:
```

Comment:
Let's state that this is a common view that we share with the Study on OSS governance at the European Commission.